exp/services/recoverysigner/internal/db/dbmigrate/migrations/20200601000000-create-signing-keys.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+
+CREATE TABLE signing_keys (
+  account_id BIGINT NOT NULL REFERENCES accounts (id) ON DELETE CASCADE,
+  id BIGINT NOT NULL PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+
+  public_key TEXT NOT NULL,
+  encrypted_secret_key BYTEA NOT NULL,
+  encryption_keyid TEXT NOT NULL
```

Comment:
Is that a typo that key ID has no separator?
```suggestion
  encryption_key_id TEXT NOT NULL
```

Reply:
https://github.com/stellar/go/pull/2645/commits/048a3b39e112bc4e0617fa238c704955c0729843
^ I feel like I've made so many mistakes programming with one eye 👁️ 🤦